tests/integration/sentinel/transitions/registration.spec.js

```diff
@@ -98,6 +98,150 @@ describe('registration', () => {
   after(() => utils.revertDb([], true));
   afterEach(() => utils.revertDb(getIds(contacts), true));
 
+  it('should add valid phone to patient doc', () => {
+    const patient_phone = '+9779841123123';
+    const patientNameAndPhone = { // has just the `patient_name`and phone so should create this person
+      _id: uuid(),
+      type: 'data_record',
+      form: 'FORM-A',
+      from: '+9779841212345',
+      fields: {
+        patient_name: 'Minerva',
+        phone_number: patient_phone
+      },
+      reported_date: moment().valueOf(),
+      contact: {
+        _id: 'person',
+        parent: { _id: 'clinic', parent: { _id: 'health_center', parent: { _id: 'district_hospital' } } }
+      }
+    };
+    const docs = [
+      patientNameAndPhone
+    ];
+    const docIds = getIds(docs);
+    let newPatientId;
+    return utils
+      .updateSettings(testForm.forms.NP, 'sentinel')
+      .then(utils.saveDocs(docs))
+      .then(() => sentinelUtils.waitForSentinel(docIds))
+      .then(() => sentinelUtils.getInfoDocs(docIds))
+      .then(infos => {
+        infos.forEach(info => {
+          chai.expect(info).to.deep.nested.include({ 'transitions.registration.ok': true });
+        });
+      })
+      .then(() => utils.getDocs(docIds))
+      .then(updated => {
+        chai.expect(updated[0].fields.phone_number).to.equal(patient_phone);
+        chai.expect(updated[0].patient_id).not.to.equal(undefined);
+        chai.expect(updated[0].tasks).to.have.lengthOf(1);
+        chai.expect(updated[0].tasks[0].messages[0]).to.include({
+          to: '+9779841212345',
+          message: `Patient Minerva (${updated[0].patient_id}) added to Clinic`
+        });
+
+        newPatientId = updated[0].patient_id;
+
+        return getContactsByReference([newPatientId, 'venus']);
+      })
+      .then(patients => {
+        chai.expect(patients.rows[0].doc).to.deep.include({
+          patient_id: newPatientId,
+          phone: patient_phone,
+          parent: { _id: 'clinic', parent: { _id: 'health_center', parent: { _id: 'district_hospital' } } },
+          name: 'Minerva',
+          type: 'person',
+          created_by: 'person',
+          source_id: patientNameAndPhone._id,
+        });
+      });
+  });
+
+  it('should not create patient from report doc when provided invalid phone', () => {
+    const patient_phone = '+97796666';
+    const patient_id =uuid();
```

Comment:
Please use camelCase for variables. 
```suggestion
    const patientPhone = '+97796666';
    const patientId = uuid();
```